README.md

```diff
@@ -28,3 +28,127 @@ After matching the slide filenames against the sample IDs from the clinical data
+
+The command
+```
+python model_train.py
+```
+will train a model using the default hyperparameters and data locations described below.
+
+The input parameters and their default values can be listed using
+```
+python model_train.py -h
+```
+that generates output
+```
+usage: model_train.py [-h] [-m M] [-p P] [-f F] [-i I] [-r R] [-t T] [-d D]
+                      [--lr LR] [--me ME] [--kp KP] [--bs BS] [--ic IC]
+                      [--ngf NGF] [--gn GN] [--ns NS] [--nm NM]
+
+Arguments for Training the SCNN/GSCNN model
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -m M        image_only or image_genome; image_only for SCNN and image_genome
+              for GSCNN. The default is image_only.
+  -p P        Path to the Train/Test patient list. The default path is ./inputs/ImageSplits.txt
+  -f F        Path to the features data. The default path is ./inputs/all_dataset.csv
+  -i I        Path to the folder containing ROI .pngs. The default path is ../images/train
+  -r R        Path where the training outputs should be generated. The default path is ./train_results
+  -t T        Path where training meta files should be generated. The default path is ./checkpoints
+  -d D        Path where binary files for training should be generated (these can be quite large and
+              should be cleaned). The default path is ./tmp
+  --lr LR     Initial learning rate. Default value = 0.001.
+  --me ME     Max number of epochs. Default value = 100.
+  --kp KP     Keeping probability for Test time dropout. Default value = 0.95.
+  --bs BS     batch size. Default value = 14.
+  --ic IC     indexes column in the number in the ./inputs/all_dataset.csv
+              file, starting from 0. Default value = 1.
+  --ngf NGF   Number of genomic features. The default is 2, considering the
+              case we have just "codeletion" and "idh mutation" as genomic
+              features.
+  --gn GN     Z Normalization of the geomic values; y for yes and n for no.
+              Default value = n.
+  --ns NS     Starting column number of Normalization for the genomic values
+              we want to do the Z normalization. This is the column number in
+              the -f argument file, starting from 0 index. Default value = 6.
+              Note: The columns we want to normalize should get assigned
+              as the last columns of the all_data.csv file.
+  --nm NM     Number of the latest models we want to save for the test time.
+              Default value = 5.
+```
+The outputs generetaed by model_train.py are:
+
+* Binary conversion of .png ROI files
+* ###_all_features.csv saved in ./train_results directory by default. ### is the date/time when the executable was run. This file contains four columns: TCGA ID (TCGA IDs of each patient), indexes (allocated index for each patient), censored (censoring status of each patient: 0 for NOT censored, 1 for censored), Survival months (survival time of each patient in months).
```

Comment:
This is just basically duplicating the input, but only for the patients used in training?

Reply:
This is used just for double checking to see if the feature values are correctly converted inside the code. Please let me know if we don't need ###_all_features.csv file as an output.